README.md

```diff
@@ -292,44 +278,45 @@ Sample walkthroughs can be found [here](docs/Walkthrough.md).
   <summary><b>Steps to run a sample from a build</b></summary>
 <br />
 
-  > :heavy_exclamation_mark: Check the [Prerequisites](#prerequisites) as there may be specific requirements for a player's computer.
-  >  Windows, for instance, requires the players to have ```The latest Microsoft Visual C++ Redistributable``` installed on their computer in order to play any distributed builds.
+  > [!NOTE] 
+  > Check the [Prerequisites](#prerequisites) as there may be specific requirements for a player's computer.
+  > For instance, Windows requires the players to have `The latest Microsoft Visual C++ Redistributable` installed on their computer in order to play any distributed builds.
 
-1. In the Unity editor menu bar, open ```File->Build Settings```.
+1. In the Unity editor menu bar, open `File->Build Settings`.
   
-    > :heavy_exclamation_mark: If you have non-sample scenes, drag a sample scene to be the 0th scene in Build Settings before you build.
+    > [!NOTE]
+    > If you have non-sample scenes, drag a sample scene to be the 0th scene in Build Settings before you build.
 
-2. Choose your desired platform, and settings, hitting ```Build``` as you normally would.
+2. Choose your desired platform, and settings, hitting `Build` as you normally would.
 
 3. Run your build.
 
-    > :heavy_exclamation_mark: A Windows build, is started by running the ```EOSBootstrapper``` application in the resulting build, and not the game application itself. it is for this and similar reasons that the ```Build And Run``` button may not always function as it usually would.
+    > [!WARNING] 
+    > A Windows build, is started by running the `EOSBootstrapper` application in the resulting build, and not the game application itself. it is for this (and similar) reasons that the `Build And Run` button may not always function as it usually would.
 
-4.  Login with a selected authentication type. <br />```Account Portal``` and then ```PersistentAuth``` is easiest for the first time. ```Dev Auth``` can be used for quicker iteration, among other options described [here](docs/Walkthrough.md), to explore features that don't require an Epic Games Account, or the options to choose for closer representation to the user experience on specific platform releases.
+4.  Login with a selected authentication type. <br />`Account Portal` and then `PersistentAuth` is easiest for the first time. `Dev Auth` can be used for quicker iteration, among other options described [here](docs/Walkthrough.md), to explore features that don't require an Epic Games Account, or the options to choose for closer representation to the user experience on specific platform releases.
```

Comment:
Similar to the line above

Reply:
Same response to the previous change request.